src/app/tokens.scss

```diff
@@ -192,18 +190,18 @@ $layout-2xl: 192px;
 
 // Typeface
 // Title copy
-$text-title-lg: 700 56px / 56px Metropolis, sans-serif;
-$text-title-md: 700 48px / 48px Metropolis, sans-serif;
-$text-title-sm: 700 38px / 40px Metropolis, sans-serif;
-$text-title-xs: 700 28px / 30px Metropolis, sans-serif;
-$text-title-2xs: 700 24px / 26px Metropolis, sans-serif;
-$text-title-3xs: 700 18px / 20px Metropolis, sans-serif;
+$text-title-lg: 700 56px / 56px var(--font-metropolis), sans-serif;
```

Comment:
Q: Should `var(--font-metropolis)` be a SCSS variable: `$font-metropolis`, etc. ?

Reply:
The variable [is defined by](https://github.com/mozilla/blurts-server/blob/c450f4535d096d0e1ed94c7d07e87d95a5e29af4/src/app/fonts/Metropolis/metropolis.ts#L25) Next.js's `localFont`, and is what allows us to use it in an SCSS variable while keeping its optimisations.

We could create a separate `$font-metropolis: var(--font-metropolis)` variable, but for that I think the question is whether we want other parts of the code to use that font independently, or if it should use all the other font settings that are included in the `$text-title-*` etc variables.